nx_parallel/algorithms/centrality/harmonic.py

```diff
@@ -0,0 +1,64 @@
+from joblib import Parallel, delayed
+import networkx as nx
+import nx_parallel as nxp
+from functools import partial
+
+__all__ = ["harmonic_centrality"]
+
+
+@nxp._configure_if_nx_active(should_run=nxp.should_run_if_sparse)
```

Comment:
I've got a question on the density-based-threshold version of `should_run`.
How do we specify that the threshold for this function should be 0.3? What is we want to change the threshold to 0.1? I see that the function itself has a default threshold of 0.3. But how do we decorate with a threshold that is specific for this function?